packages/services/api/src/modules/collection/module.graphql.ts

```diff
@@ -0,0 +1,154 @@
+import { gql } from 'graphql-modules';
+
+export const typeDefs = gql`
+  type DocumentCollection {
+    id: ID!
+    name: String!
+    description: String
+    createdAt: DateTime!
+    updatedAt: DateTime!
+    createdBy: User!
+    operations(first: Int, after: String): DocumentCollectionOperationsConnection!
+  }
+
+  type DocumentCollectionConnection {
+    nodes: [DocumentCollection!]!
+    total: Int!
+  }
+
+  type DocumentCollectionOperationsConnection {
+    nodes: [DocumentCollectionOperation!]!
+    total: Int!
+  }
```

Comment:
https://relay.dev/graphql/connections.htm 👀

Reply:
please be more descriptive, do you mean `total` field?

Reply:
Please follow the connection specification for naming and implementing Connection fields and types as we already do for all new fields added.

`nodes` should be `edges` and the `PageInfo` field should be added.

Reply:
I am not sure if it was me or @dotansimha who made yesterday refactoring previously we had `OperationEdge`. This is a nice time to extend `graphql-eslint` relay config hehe